inclusion_connect/middleware.py

```diff
@@ -9,3 +11,19 @@ def middleware(request):
         return response
 
     return middleware
+
+
+def limit_staff_users_to_admin(get_response):
+    def middleware(request):
+        user = request.user
+
+        if user.is_staff and not request.path.startswith("/admin/") and not request.path == "/403/":
```

Comment:
Pour autant que je sache, il n’y a pas de page à l’URL `/403/`.
```suggestion
        user = request.user

        if user.is_staff and not request.path.startswith("/admin/"):
```

Reply:
idem